pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -29,11 +30,33 @@ import (
 )
 
 const (
-	TestCertsTmp = "./test-certs-tmp"
+	TestCertsTmp   = "./test-certs-tmp"
+	TestCaCertPath = "./test-ca.crt"
+	TestCaKeyPath  = "./test-ca.key"
 )
 
+func writeMockCert() {
+	certData := []byte("-----BEGIN CERTIFICATE-----\nMIIB6TCCAY6gAwIBAgIUQlTeCuvulLnDo9JWIAIJmMQB0jUwCgYIKoZIzj0EAwIw\nQDELMAkGA1UEBhMCQ04xCzAJBgNVBAgTAkdEMQ8wDQYDVQQHEwZGb1NoYW4xEzAR\nBgNVBAMTCkt1YmVybmV0ZXMwHhcNMjQwNTA4MDg1MTAwWhcNMjkwNTA3MDg1MTAw\nWjBAMQswCQYDVQQGEwJDTjELMAkGA1UECBMCR0QxDzANBgNVBAcTBkZvU2hhbjET\nMBEGA1UEAxMKS3ViZXJuZXRlczBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABLz2\n4B6I1ymHkD8VYNbYm7J1PkPRKlwIPx1GzFm92EyK5nkNY4mMmIDwTocKM08IBNBB\nnesvfH6ipBqVU95IohCjZjBkMA4GA1UdDwEB/wQEAwIBBjASBgNVHRMBAf8ECDAG\nAQH/AgECMB0GA1UdDgQWBBQTkWmoVW9/Cb/bl4rHsCGKJiHbSzAfBgNVHSMEGDAW\ngBQTkWmoVW9/Cb/bl4rHsCGKJiHbSzAKBggqhkjOPQQDAgNJADBGAiEA16jlnBod\nF9Ba4VZ/CGdaROZTaF988AYjYKdgOsun2+0CIQCRkpzQ0+XeSk6qZJ+fwYBdoXUL\nSmz0jaVWUjrO2GsIrw==\n-----END CERTIFICATE-----")
+	if err := os.MkdirAll(filepath.Dir(TestCaCertPath), os.FileMode(0755)); err != nil {
+		fmt.Println(err)
+	}
+	if err := os.WriteFile(TestCaCertPath, certData, os.FileMode(0644)); err != nil {
+		fmt.Println(err)
+	}
+	keyData := []byte("-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIIPimXOHhF5n74QbTprxcvLE/WURZL3w4IuhY6B/LfufoAoGCCqGSM49\nAwEHoUQDQgAEvPbgHojXKYeQPxVg1tibsnU+Q9EqXAg/HUbMWb3YTIrmeQ1jiYyY\ngPBOhwozTwgE0EGd6y98fqKkGpVT3kiiEA==\n-----END EC PRIVATE KEY-----")
+	if err := os.MkdirAll(filepath.Dir(TestCaKeyPath), os.FileMode(0755)); err != nil {
+		fmt.Println(err)
+	}
+	if err := os.WriteFile(TestCaKeyPath, keyData, os.FileMode(0644)); err != nil {
+		fmt.Println(err)
+	}
+}
+
```

Comment:
I think it can Use `GenCerts` to generate cert files. see https://github.com/karmada-io/karmada/blob/625d8a5b779ee7f97ab03ff86239614a58157c69/pkg/karmadactl/cmdinit/cert/cert.go#L264-L271

Reply:
This is the same as what I think here https://github.com/karmada-io/karmada/pull/5127#discussion_r1670454248

Reply:
Thanks, your version is better. @chaosi-zju @liangyuanpeng